packages/wonder-blocks-birthday-picker/src/components/__tests__/birthday-picker.test.js

```diff
@@ -346,6 +358,68 @@ describe("BirthdayPicker", () => {
             // Assert
             expect(onChange).toHaveBeenCalledTimes(1);
         });
+
+        it("onChange triggers the first day of the month when monthYearOnly is set", async () => {
+            // Arrange
+            const onChange = jest.fn();
+
+            // Act
+            render(<BirthdayPicker monthYearOnly={true} onChange={onChange} />);
```

Comment:
This seems like part of the `Arrange` and the stuff after it is the actual action being tested. What do you think?